### PR TITLE
A couple of fixes for building against the latest libraries.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: moonsla.go [hunk 1]
```diff
@@ -164,11 +164,12 @@ func main() {
...
 	}
-	api := slack.New(slackToken)
-
 	logger := log.New(os.Stdout, "slack-bot: ", log.Lshortfile|log.LstdFlags)
-	slack.SetLogger(logger)
-	api.SetDebug(false)
+
+	api := slack.New(
+		slackToken,
+		slack.OptionDebug(true),
```
Comment (Crazybus) [inline moonsla.go]: Could you switch this back to the default of false? Making it configurable would also be ok. 

With this enabled I'm getting spammed by logs:

```
slack-bot: 2019/10/03 08:39:10 websocket_managed_conn.go:326: Sending PING  1
slack-bot: 2019/10/03 08:39:10 websocket_managed_conn.go:369: Incoming Event: {"type":"pong","timestamp":1570084750,"reply_to":1}
slack-bot: 2019/10/03 08:39:40 websocket_managed_conn.go:326: Sending PING  2
slack-bot: 2019/10/03 08:39:40 websocket_managed_conn.go:369: Incoming Event: {"type":"pong","timestamp":1570084780,"reply_to":2}

```

```suggestion
		slack.OptionDebug(false),
```

**Review by Crazybus — CHANGES_REQUESTED** [review]
The rest LGTM! Thank you for fixing this up :) I really should get around to switching it over to use go modules instead of the old "get the latest of everything" go used to use.

File: moonsla.go (hunk 1 above)
Comment (ghost) [reply moonsla.go]: Oops! Sorry. Fixed in 4d179b2

**Review by Crazybus — APPROVED** [review]
LGTM!

**Pull request merged**